app/models/product.rb

```diff
@@ -0,0 +1,2 @@
+class Product < ApplicationRecord
```

Comment:
Hey Hadi! Code looks good! Just wondering, is there any reason why the product model file is empty? Were you trying to do TDD? Just curious